gradle/libs.versions.toml

```diff
@@ -115,6 +116,7 @@ compose-typedNavigation = "com.kiwi.navigation-compose.typed:core:0.10.0"
 ktlint-core = { module = "com.pinterest.ktlint:ktlint-cli", version.ref = "ktlint-core" }
 ktlint-gradlePlugin = { module = "io.github.usefulness:ktlint-gradle-plugin", version.ref = "ktlint-gradlePlugin" }
 ktlint-compose = "io.nlopez.compose.rules:ktlint:0.4.22"
+datastore-core-android = { group = "androidx.datastore", name = "datastore-core-android", version.ref = "datastoreCoreAndroid" }
```

Comment:
You should be able to use the already existing data store dependency:
```
datastore = "androidx.datastore:datastore-preferences:1.1.4"
```

(To use it, just call `implementation(libs.datastore)`)